### PR TITLE
Fix for #104 - Added new section Sharpen your Skills

### LINKED ISSUE #104: Insert a new section containing list of Programing Language
I would like to introduce a section named "Sharpen you skills" below About us section, which will include a list of different programming languages for user to make easy choice for practise.

As a GSSOC'21 participant I would like to work on this issue .Please assign me this task.

A brief idea is given in picture.
Capture

### PR DESCRIPTION
**Pull request by neha-baisane**
Fixes #104 

This is  new pull request for issue.104. Made all the changes @harikesh409 Please review.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -264,7 +264,62 @@ hr {
...
+/******************Section-Sharpen your skills*****************/
+.section table{
+  border-collapse: collapse;
+  border-spacing: 0;
+  width: 90%;
+  border: 1px solid #ddd;
+  align: centre;
+}
+
+center {
```
Comment (harikesh409) [inline css/style.css:276]: `center` tag by default provides centering so no need to give margin left and margin right auto.

File: index.html [hunk 2]
```diff
@@ -165,6 +163,34 @@ <h1 class="text-center">About Us</h1>
     </section>
     <!-- about ends -->
 
+    <!--Section-Sharpen your skills starts-->
+<section id="sharpen-skills" class="section">
```
Comment (harikesh409) [inline index.html:167]: Make the class name `skills-section`.

File: css/style.css [hunk 3]
```diff
@@ -264,7 +264,62 @@ hr {
     margin-top: 50px;
 }
 
+/******************Section-Sharpen your skills*****************/
+.section table{
```
Comment (harikesh409) [inline css/style.css:268]: In similar way change other tags also not only table.

- th
- tr
- td
- a
- hr
- h1

File: css/style.css [hunk 4]
```diff
@@ -264,7 +264,62 @@ hr {
...
+
+th, td {
+  text-align: left;
+  padding: 30px;
+  font-weight: bold;
+  font-size:20px;
+  text-shadow: 2px 2px #0000;
+}
+a:Link{
+   color:#000000;
```
Comment (harikesh409) [inline css/style.css:294]: If the color is same in all states then no need to specify it individually.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by neha-baisane** [comment]
@harikesh409 Done with all the changes . Please have a look.

**Review by harikesh409 — APPROVED** [review]

**Comment by neha-baisane** [comment]
@ravikr126 @harikesh409  Done from my side and if everything is fine please merge my PR.

**Comment by ravikr126** [comment]
Please remove conflict then we review and merge it @neha-baisane

**Comment by neha-baisane** [comment]
@ravikr126 Please guide me how to resolve this conflict.

**Comment by ravikr126** [comment]
You need to pull new code and make change on it .Some one already changes same line that you edited.

You can read this also
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-merge-conflicts#:~:text=If%20your%20merge%20conflict%20is,GitHub%20using%20the%20conflict%20editor.&text=push%20the%20change.-,For%20more%20information%2C%20see%20%22Resolving%20a%20merge,conflict%20on%20the%20command%20line.%22

**Comment by neha-baisane** [comment]
@ravikr126 resolved the conflicts . Please review.

**Comment by neha-baisane** [comment]
@ravikr126 resolved the conflicts . Please review.

**Comment by ravikr126** [comment]
Then create new pr request

**Comment by harikesh409** [comment]
Moved to #322

**Pull request closed without merging**